tools/settings.py

```diff
@@ -75,6 +75,16 @@ class CustomLogLevels:
     os.getenv("CURRENCY_CONVERTER_EXTRAPOLATION_MAX_DAYS_FALLBACK", 180)
 )
 
+FIELD_COVERAGE_STEP = "field_coverage"
+FIELD_QUALITY_STEP = "field_quality"
+DATASET_STEP = "dataset"
+TIME_BASED_STEP = "time_based"
+REPORT_STEP = "report"
```

Comment:
The project uses this pattern elsewhere, which would help keep the constants organized. The class can go near the top of the file. We're also missing compiled_release / data_item.

```
class Steps:
    FIELD_COVERAGE = "field_coverage"
    FIELD_QUALITY = "field_quality"
    COMPILED_RELEASE = "compiled_release"
    DATASET = "dataset"
    TIME_BASED = "time_based"
    REPORT = "report"